homeassistant/components/hunterdouglas_powerview/cover.py

```diff
@@ -119,10 +119,9 @@ def device_state_attributes(self):
     @property
     def supported_features(self):
         """Flag supported features."""
-        supported_features = SUPPORT_OPEN | SUPPORT_CLOSE | SUPPORT_SET_POSITION
         if self._device_info[DEVICE_MODEL] != LEGACY_DEVICE_MODEL:
-            supported_features |= SUPPORT_STOP
-        return supported_features
+            return SUPPORT_STOP | SUPPORT_OPEN | SUPPORT_CLOSE | SUPPORT_SET_POSITION
+        return SUPPORT_OPEN | SUPPORT_CLOSE | SUPPORT_SET_POSITION
```

Comment:
When the same thing is repeated like this it is harder to read the code because both lines must now be compared to see how they differ.

It is also harder to maintain the code when one has to remember to change something in multiple places.

Reply:
I can definitely see where you are coming from, I thought that three/four would be the maximum number of support features where you could still easily differentiate between two return statements. I will revert this change here for code readability.

Reply:
I agree with @amelchio. I don't think that this is an improvement.

Reply:
Yeah I understand and have reverted this change.